robotidy/transformers/SplitTooLongLine.py

```diff
@@ -0,0 +1,112 @@
+from itertools import tee
+
+from robot.api.parsing import (
+    ModelTransformer,
+    Token
+)
+
+
+def eol():
+    return Token(Token.EOL)
+
+
+class SplitTooLongLine(ModelTransformer):
+    def __init__(self, line_length: int = 120):
+        self.line_length = line_length
+
+    def visit_KeywordCall(self, node):  # noqa
+        if all(token.end_col_offset <= self.line_length for token in node.tokens[::-1]):
+            return node
+        if not node.get_tokens(Token.ARGUMENT):  # return if there are no arguments - nothing to split
+            return node
+        return self.split_keyword_call(node)
+
+    def split_keyword_call(self, node):
+        is_single_line = node.lineno == node.end_lineno
+
+        indent = node.tokens[0]
+        assignment = list(self.insert_seperator(node.get_tokens(Token.ASSIGN)))
+        keyword = node.get_token(Token.KEYWORD)
+
+        if is_single_line:
```

Comment:
I think that if any of the lines is going above the limit, we can reshuffle whole keyword so there is no need to take into account previous lines. Just take data tokens and comments and insert separators/new lines

Reply:
It's not taking into account previous lines, rather it's about comment placement. It seems for now to be a good idea to make the comment placement strategy differentiate between whether we have a call with `...` continuation or not, hence `if is_single_line` - it means "the keyword call has no second-line continuation", therefore "the comment from the first line should land above it"